### PR TITLE
Update Wink lamp test to use the Hub Translator

### PR DESCRIPTION
**Pull request by begillil**
@arjun-msft, I was trying out your latest changes and realized the Wink Lamp tests and package.json never got updated.

### CODE REVIEW

**Comment by arjun-msft** [comment]
Ha I have the package.json in review as well. Thanks for updating the tests file!

File: org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/tests/test.js [hunk 1]
```diff
@@ -3,66 +3,84 @@ var test = require('ava');
 var OpenT2T = require('opent2t').OpenT2T;
 var config = require('./testConfig');
 
-console.log('Device Under Test -  Name: ' + config.Device.name + '  Props: ' + JSON.stringify(config.Device.props));
+var config = require('./testConfig');
```
Comment (arjun-msft) [inline org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/tests/test.js]: is this in there twice?

**Comment by arjun-msft** [comment]
:shipit:

File: org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/tests/test.js (hunk 1 above)
Comment (begillil) [inline org.opent2t.sample.lamp.superpopular/com.wink.lightbulb/js/tests/test.js]: Good catch. Removed.

---
In reply to: [78666577](https://github.com/openT2T/translators/pull/95#discussion_r78666577) [](ancestors = 78666577)

**Pull request merged**